html/elements/input.json

```diff
@@ -0,0 +1,95 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/input",
+          "description": "Basic support",
+          "support": {
+            "chrome": {
+              "version_added": "1"
```

Comment:
Please add `chrome_android` and `webview_android`. Use `true` for their values.

Reply:
done in f8350e3